Marlin/planner.cpp

```diff
@@ -522,7 +522,9 @@ void Planner::check_axes_activity() {
 }
 
 #if PLANNER_LEVELING
-
+  /**
+   * lx, ly, lz - cartesian positions in mm
```

Comment:
ok, I'll fix it.
I've would underline that the coordinates are NOT delta or scara, that are x/y/z